packages/components/card/card.scss

```diff
@@ -54,20 +53,18 @@ $card-border-hover-color: $color_nhsuk-grey-3;
 .nhsuk-card--clickable {
   border-bottom-width: $card-border-bottom-width;
 
-  &:hover,
-  &:active {
-    cursor: pointer;
-
-    .nhsuk-card__heading a,
-    .nhsuk-card__link {
-      color: $nhsuk-link-hover-color;
-      text-decoration: none;
-
-      &:focus {
-        color: $nhsuk-focus-text-color;
-      }
+  .nhsuk-card__heading a,
+  .nhsuk-card__link {
```

Comment:
@chrimesdev I've put both these selectors in here in order to be consistent with what came before. Is this still necessary? I assume it's backwards compatibility before the `.nhsuk-card__link` class was added